### PR TITLE
Remove redundant class attributes declarations

### PR DESCRIPTION
**Pull request by alpinus4**
There are many redundant class attributes declarations that are inherited anyways. 
It greatly clutters the code and makes it less readable.

### CODE REVIEW

File: src/actions/action.py [hunk 1]
```diff
@@ -6,8 +6,6 @@ class Action(Serializable):
     Represents abstract object responsible for interacting
     with a `scene`.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/action.py:10]: Remove `Properties` import.

File: src/actions/attacks/slash.py [hunk 2]
```diff
@@ -13,8 +13,6 @@ class Slash(Action):
     """
     Simple action for attacking.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/attacks/slash.py:17]: Remove `Properties` import.

File: src/actions/deal_dmg.py [hunk 3]
```diff
@@ -10,8 +10,6 @@ class DealDmg(Action):
     """
     Simple action for dealing damage.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/deal_dmg.py:14]: Remove `Properties` import.

File: src/actions/destroy.py [hunk 4]
```diff
@@ -10,8 +10,6 @@ class Destroy(Action):
     """
     Simple action for dying.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/destroy.py:14]: Remove `Properties` import.

File: src/actions/idle.py [hunk 5]
```diff
@@ -7,8 +7,6 @@ class Idle(Action):
     """
     Simple action for idling.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/idle.py:11]: Remove `Properties` import.

File: src/actions/move.py [hunk 6]
```diff
@@ -10,8 +10,6 @@ class Move(Action):
     Simple action for changing position of `Agent`. It does not log anything
     in case the movement was not possible(destination is not walkable etc).
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/move.py:14]: Remove `Properties` import.

File: src/actions/nearby_objects.py [hunk 7]
```diff
@@ -9,8 +9,6 @@ class NearbyObjects(Action):
     """
     Action to retrieve nearby objects within the range of the triggering object
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/nearby_objects.py:13]: Remove `Properties` import.

File: src/actions/next_scene.py [hunk 8]
```diff
@@ -7,8 +7,6 @@ class NextScene(Action):
     """
     `Action` used to create next scene. Called on collision by `Portal`.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/next_scene.py:11]: Remove `Properties` import.

File: src/actions/use.py [hunk 9]
```diff
@@ -9,8 +9,6 @@ class Use(Action):
     """
     An action that provides the opportunity to interact with another object
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/use.py:13]: Remove `Properties` import.

File: src/actions/wait_for_code.py [hunk 10]
```diff
@@ -7,9 +7,7 @@ class WaitForCode(Action):
     Action blocking execution of the runtime until it reads
     message from `stdin` saying that the `code` was uploaded
     """
-    base: str
     interactive: bool
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/wait_for_code.py:12]: Remove `Properties` import.

File: src/actions/wave.py [hunk 11]
```diff
@@ -6,8 +6,6 @@ class Wave(Action):
     """
     Simple action for playing an animation on frontend.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/actions/wave.py:10]: Remove `Properties` import.

File: src/objects/agent.py [hunk 12]
```diff
@@ -18,10 +18,6 @@ class Agent(Object):
     """
     Simple agent existing in a `scene`.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/objects/agent.py:22]: Remove `Properties` import.

File: src/objects/floor.py [hunk 13]
```diff
@@ -7,10 +7,6 @@ class Floor(Object):
     """
     Simple walkable `Object`.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/objects/floor.py:11]: Remove `Properties` import.

File: src/objects/interactive_object.py [hunk 14]
```diff
@@ -7,8 +7,6 @@ class InteractiveObject(Object):
     """
     Abstract represents object that can interact with each other
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/objects/interactive_object.py:11]: Remove `Properties` import.

File: src/objects/npcs/enemies/training_dummy.py [hunk 15]
```diff
@@ -12,10 +12,6 @@ class TrainingDummy(Enemy):
     """
     Enemy that does not do anything.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/objects/npcs/enemies/training_dummy.py:16]: Remove `Properties` import.

File: src/objects/object.py [hunk 16]
```diff
@@ -7,8 +7,6 @@ class Object(Serializable):
     Abstract object existing in a `scene`. Represents part of scene's state,
     in contrast to `Action` which represents only modifications of the state.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/objects/object.py:11]: Remove `Properties` import.

File: src/objects/portal.py [hunk 17]
```diff
@@ -8,10 +8,6 @@ class Portal(Object):
     """
     `Object` allowing the player to move to a next `scene`.
     """
-    base: str
-    properties: Properties
```
Comment (filip-szweda) [inline src/objects/portal.py:12]: Remove `Properties` import.

**Review by filip-szweda — CHANGES_REQUESTED** [review]

**Review by blazej-smorawski — APPROVED** [review]

**Review by Krystian030 — APPROVED** [review]

**Pull request merged**